eng/common/pipelines/templates/steps/bypass-local-dns.yml

```diff
@@ -8,7 +8,9 @@ steps:
           succeededOrFailed(),
           or(
             eq(variables['OSVmImage'], 'ubuntu-18.04'),
-            eq(variables['OSVmImage'], 'MMSUbuntu18.04')
+            eq(variables['OSVmImage'], 'ubuntu-20.04'),
```

Comment:
Lets get things unblocked as possible but it might be worth switching this to something like `contains(variables['OSVmImage'], 'ubunutu')` (not sure if it is case sensitive) in a future follow-up.